pkg/controller/iotconfig/project_operator.go

```diff
@@ -1,84 +0,0 @@
-/*
- * Copyright 2019, EnMasse authors.
- * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
- */
-
-package iotconfig
-
-import (
-	"context"
-
-	"github.com/enmasseproject/enmasse/pkg/util/images"
-
-	iotv1alpha1 "github.com/enmasseproject/enmasse/pkg/apis/iot/v1alpha1"
-	"github.com/enmasseproject/enmasse/pkg/util/install"
-	appsv1 "k8s.io/api/apps/v1"
-	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/resource"
-)
-
-func (r *ReconcileIoTConfig) processProjectOperator(ctx context.Context, config *iotv1alpha1.IoTConfig) error {
-	return r.processDeployment(ctx, "iot-operator", config, false, r.reconcileProjectOperator)
-}
-
-func (r *ReconcileIoTConfig) reconcileProjectOperator(config *iotv1alpha1.IoTConfig, deployment *appsv1.Deployment) error {
-
-	install.ApplyDeploymentDefaults(deployment, "iot", deployment.Name)
-
-	applyDefaultDeploymentConfig(deployment, config.Spec.ServicesConfig.Operator.ServiceConfig, nil)
```

Comment:
For the upgrade case, I see nothing that will remove these resource (iot-operator deployment, service account etc) so they will be orphaned.